### ISSUE TITLE
google_contacts plugin shows "Object id #xxx" as contact name

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#94]
```
The google_contacts plugin imported all my contacts email addresses properly 
but their names were "Object id #xxx" where xxx was a number. I had to change 
line 328 of google_contacts.php from:

$obj->name = (string) $entry->title;

to: 

$obj->name = (string) $entry->title->text;

to have the plugin show the proper contact names.

I'm on PHP 5.1.6, RoundCube 0.4-beta.
```

Original issue reported on code.google.com by `bacob...@gmail.com` on 12 Aug 2010 at 6:56

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 13 Aug 2010 at 5:23
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3878.1.
```

Original comment by `rol...@liebl.ath.cx` on 14 Aug 2010 at 4:47
- Changed state: **Fixed**
